webui/react/src/pages/ExperimentDetails/ExperimentMultiTrialTabs.tsx

```diff
@@ -120,9 +119,6 @@ const ExperimentMultiTrialTabs: React.FC<Props> = ({
       </TabPane>
       {showExperimentArtifacts ? (
         <>
-          <TabPane key={TabType.Checkpoints} tab="Checkpoints">
```

Comment:
just minor thing: i think its nice to remove `Checkpoint` from `TabType`

Reply:
Great catch thanks!